simplewfs/SimpleWfsApi.php

```diff
@@ -54,12 +46,12 @@ public function readRequest(): void
             if (isset($this->apiRequest->getVar['outputFormat'])) {
                 $this->outputFormat = $this->apiRequest->getVar['outputFormat'];
             }
-            if (isset($this->apiRequest->getVar['token'])) {
-                $this->token = $this->apiRequest->getVar['token'];
-                $this->user->setToken($this->token);
-                $this->user->getUserFromToken();
-                $this->user->checkToken();
-            } //else {
+            // if (isset($this->apiRequest->getVar['token'])) {
+            //     $this->token = $this->apiRequest->getVar['token'];
+            //     $this->user->setToken($this->token);
+            //     $this->user->getUserFromToken();
+            //     $this->user->checkToken();
+            // } //else {
             //$this->token = '';
             //}
```

Comment:
can we just remove this commented code if you don't need it?
if you want to keep it for debugging maybe comment it?

Reply:
removed.